workers/generic-worker/schemas/multiuser_posix.yml

```
@@ -1,369 +1,604 @@
+              Conventionally (although not enforced) path elements are forward slash separated. Example:
+              `public/build/a/house`. Note, no scopes are required to read artifacts beginning `public/`.
+              Artifact names not beginning `public/` are scope-protected (caller requires scopes to
+              download the artifact). See the Queue documentation for more information.
+
+              Since: generic-worker 8.1.0
+          expires:
+            title: Expiry date and time
+            type: string
+            format: date-time
+            description: |-
+              Date when artifact should expire must be in the future, no earlier than task deadline, but
+              no later than task expiry. If not set, defaults to task expiry.
+
+              Since: generic-worker 1.0.0
+          contentType:
+            title: Content-Type header when serving artifact over HTTP
+            type: string
+            description: |-
+              Explicitly set the value of the HTTP `Content-Type` response header when the artifact(s)
+              is/are served over HTTP(S). If not provided (this property is optional) the worker will
+              guess the content type of artifacts based on the filename extension of the file storing
+              the artifact content. It does this by looking at the system filename-to-mimetype mappings
+              defined in multiple `mime.types` files located under `/etc`. Note, setting `contentType`
+              on a directory artifact will apply the same contentType to all files contained in the
+              directory.
+
+              See [mime.TypeByExtension](https://pkg.go.dev/mime#TypeByExtension).
+
+              Since: generic-worker 10.4.0
+          contentEncoding:
+            title: Content-Encoding header when serving artifact over HTTP.
+            type: string
+            enum:
+            - identity
+            - gzip
+            description: |-
+              Content-Encoding for the artifact. If not provided, `gzip` will be used, except for the
+              following file extensions, where `identity` will be used, since they are already
+              compressed:
+
+              * 7z
+              * bz2
+              * deb
+              * dmg
+              * flv
+              * gif
+              * gz
+              * jpeg
+              * jpg
+              * png
+              * swf
+              * tbz
+              * tgz
+              * webp
+              * whl
+              * woff
+              * woff2
+              * xz
+              * zip
+              * zst
+
+              Note, setting `contentEncoding` on a directory artifact will apply the same content
+              encoding to all the files contained in the directory.
+
+              Since: generic-worker 16.2.0
+        required:
+        - type
+        - path
+    features:
+      title: Feature flags
```

```suggestion
      title: Docker Worker feature flags
```

This will avoid the go type getting named `FeatureFlags1`.

---

Nice catch! Thank you 🎉